cargo-shuttle/src/args.rs

```diff
@@ -325,7 +325,7 @@ impl InitTemplateArg {
             Tide => "tide/hello-world",
             Tower => "tower/hello-world",
             Warp => "warp/hello-world",
-            None => "custom/none",
+            None => "custom-service/none",
```

Comment:
Not sure how the change relates to this PR?

Reply:
It doesn't. It's a fix required for the tests to pass and a regression that came with custom-resources. I modified `shuttle-examples`, by changing the `custom` directory name to `custom-service`. The PR is merged on `shuttle-examples` main and will cause failure to initing a `none` example with `cargo-shuttle`.

Reply:
Ah, I see. This should ideally have been a separate PR